app/src/main/res/layout/card_links_box.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
```

Comment:
Really similar refactoring is also done by @friedger in PR #416 
This will cause conflicts a lot.